poetry/masonry/builders/wheel.py

```diff
@@ -181,9 +181,11 @@ def _write_metadata(self, wheel):
             with self._write_to_zip(wheel, self.dist_info + "/entry_points.txt") as f:
                 self._write_entry_points(f)
 
-        for base in ("COPYING", "LICENSE"):
-            for path in sorted(self._path.glob(base + "*")):
-                self._add_file(wheel, path, "%s/%s" % (self.dist_info, path.name))
+        for license_file_name in self._package.license_files:
+            license_file = self._path / license_file_name
+            self._add_file(
+                wheel, license_file, "%s/%s" % (self.dist_info, license_file.name)
```

Comment:
`license_file.name` is `license_file_name` and if the latter was used here, the `license_file` symbol wouldn't have to be allocated.

Reply:
While I'll admit the names would imply that, the step is needed. Otherwise `_add_file()` will raise `FileNotFoundError`.  So I've renamed them in the next push to be more clear what their purpose is.